### ISSUE TITLE
`noname-attribute` creates two defects instead of one

### ISSUE THREAD

**Issue by volodya-lombrozo** [objectionary/lints#451]
I have the following EO program:

```eo
+package f
# No comments.
  [] > main
    QQ.io.stdout
         "Hello world"
```

When I run:

```java
 final Collection<Defect> defects = new Program(xmir).without(
                "unlint-non-existing-defect",
                "empty-object",
                "sprintf-without-formatters"
            ).defects();
```
It gives me plenty number of errors. However, what is the most important, the defect `noname-attribute` is generated twice:

```
[foo.x.main noname-attribute ERROR]:5 The object has an attribute without a name
[foo.x.main noname-attribute ERROR]:5 The object has an attribute without a name
```

I thing we need to have only one defect related to this issue

**Comment by volodya-lombrozo** [comment]
@h1alexbel What do you think?

**Comment by h1alexbel** [comment]
@volodya-lombrozo looks like a bug, I will take it. Thanks for reporting

**Comment by yegor256** [comment]
@maxonfjvipon why do we even allow such attributes, syntax wise?

```
[] > foo
  x
```

This should be illegal, I believe.

**Comment by maxonfjvipon** [comment]
@yegor256 because we can't tell the difference between objects which must be named and must not. Maybe we can, but I'm not sure how to properly integrate it into our grammar without making it so verbose

**Comment by h1alexbel** [comment]
@yegor256 can we release new version? It should fix problems in EO: https://github.com/objectionary/eo/pull/4063

**Comment by 0crat** [comment]
@volodya-lombrozo Hey there, bug hunter! 🐛 Awesome job spotting that new bug - you've just bagged yourself 8 points! Your sharp eye is helping us level up our project quality. Keep 'em coming! If you spot anything else in the repo that [seems off](https://www.yegor256.com/2018/02/06/where-to-find-more-bugs.html), [don't hold back](https://www.yegor256.com/2014/04/13/bugs-are-welcome.html) - [report away](https://www.yegor256.com/2018/04/24/right-way-to-report-bugs.html)! Your current score is [+82](https://www.eolang.org/zerocracy/objectionary-vitals.html). Keep up the great work!

**Comment by yegor256** [comment]
@rultor release, tag is `0.0.44`

**Comment by volodya-lombrozo** [comment]
@h1alexbel Thanks!
